src/main/java/uk/gov/hmcts/darts/usermanagement/service/validation/UserEmailValidator.java

```diff
@@ -10,12 +11,19 @@
 
 @Component("duplicateEmailValidator")
```

Comment:
As you changed the name of the class, do you think the component name should also change